tests/Resources/ti.locale.test.js

```diff
@@ -44,6 +44,12 @@ describe('Titanium.Locale', () => {
 			});
 		});
 
+		describe.android('.applicationLocales', () => {
+			it('is a Array', () => {				
```

Comment:
```suggestion
			it('is a Array', () => {
```

removing some spaces at the end :)